.github/workflows/ci.yml

```diff
@@ -28,23 +41,25 @@ jobs:
         run: nix develop --command treefmt --fail-on-change
 
       - name: Clippy
+        if: steps.filter.outputs.rust == 'true'
         run: nix develop --command cargo clippy --all-targets -- -D warnings
 
       - name: Repository cleanliness
+        if: steps.filter.outputs.rust == 'true'
         run: git diff --exit-code
 
       - name: Test
+        if: steps.filter.outputs.flake == 'true' && steps.filter.outputs.rust == 'true'
```

Comment:
Wouldn't this only run if both rust and nix files have been changed? Maybe it should be
```nix
if: steps.filter.outputs.flake == 'true' || steps.filter.outputs.rust == 'true'
```

Reply:
Ah, yes. Good catch.